packages/widgets-mitosis/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "widgets-mitosis",
+  "version": "1.0.0",
```

Comment:
no version here, and you can make this private. We will publish the generated sdks only

Reply:
Similar to the packages/sdks package.json